src/Typography/Typography.component.tsx

```diff
@@ -91,10 +91,10 @@ type DisplayValues =
 
 export interface TypographyProps {
     className?: string;
-    as?: Elements;
+    tag?: Elements;
     children?: any;
     weight?: FontWeights;
-    color?: Colors;
+    color?: 'inherit' | Colors;
```

Comment:
After discussion with @sharkparty, decided it was best to default Typography to inherit